### ISSUE TITLE
Color escape sequences are not correctly shown in terminal

### ISSUE THREAD

**Issue by EmilySeville7cfg** [DannyBen/bashly#396]
### Bashly Version

Edge Ruby Gem

### Description

![image](https://github.com/DannyBen/bashly/assets/42812113/8593e94e-26d6-48ba-993c-155f1bd8601c)

### Reproduction Steps

- `bashly shell`

### Actual Behavior

Escape color codes shown instead of actual colors.

### Expected Behavior

Correctly colored **bashly** prompt.

**Comment by DannyBen** [comment]
This is weird. Not sure how to reproduce it, as we are both on Ubuntu.

Maybe you can send me the output of your `env` command **minus any secrets** or positively unrelated vars?

**Comment by DannyBen** [comment]
@EmilySeville7cfg I am guessing that if you run the below command you get the same unescaped codes?

```bash
$ ruby -r readline -e 'Readline.readline "\n\e[33m\e[1mbashly\e[0m > "'
```

Bashly terminal is using Ruby's build in `Readline` behind the scenes, which simply has a `prompt` as a first argument.

What does your `ruby -v` say?

**Comment by EmilySeville7cfg** [comment]
> Maybe you can send me the output of your env command minus any secrets or positively unrelated vars?

```
CLUTTER_IM_MODULE=ibus
CMD_DURATION_MS=22
COLORTERM=truecolor
COMPIZ_CONFIG_PROFILE=mate
DBUS_SESSION_BUS_ADDRESS=unix:path=/run/user/1000/bus,guid=b862924d49ef2ebbbe08971f64957d55
DBUS_STARTER_ADDRESS=unix:path=/run/user/1000/bus,guid=b862924d49ef2ebbbe08971f64957d55
DBUS_STARTER_BUS_TYPE=session
DESKTOP_SESSION=mate
DISPLAY=:0
ENV=/home/emilyseville7cfg/.dashrc
GDMSESSION=mate
GEM_HOME=/home/emilyseville7cfg/.gem
GPG_AGENT_INFO=/run/user/1000/gnupg/S.gpg-agent:0:1
GTK_IM_MODULE=ibus
GTK_MODULES=gail:atk-bridge:gail:atk-bridge:canberra-gtk-module
GTK_OVERLAY_SCROLLING=0
HOME=/home/emilyseville7cfg
IM_CONFIG_PHASE=1
INVOCATION_ID=cd135c665a12414db4e99e3d0de39f76
JOURNAL_STREAM=8:32276
LANG=en_US.UTF-8
LANGUAGE=en_US:en
LAST_EXIT_CODE=0
LC_ADDRESS=en_CA.UTF-8
LC_IDENTIFICATION=en_CA.UTF-8
LC_MEASUREMENT=en_CA.UTF-8
LC_MONETARY=en_CA.UTF-8
LC_NAME=en_CA.UTF-8
LC_NUMERIC=en_CA.UTF-8
LC_PAPER=en_CA.UTF-8
LC_TELEPHONE=en_CA.UTF-8
LC_TIME=en_CA.UTF-8
LOGNAME=emilyseville7cfg
MANAGERPID=2026
MATE_DESKTOP_SESSION_ID=this-is-deprecated
PAPERSIZE=letter
PATH=/home/emilyseville7cfg/.cargo/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin:/home/emilyseville7cfg/.cargo/bin
PWD=/home/emilyseville7cfg
QT_ACCESSIBILITY=1
QT_FONT_DPI=96
QT_IM_MODULE=ibus
QT_QPA_PLATFORMTHEME=gtk2
QT_SCALE_FACTOR=1
SESSION_MANAGER=local/emily-pc:@/tmp/.ICE-unix/2241,unix/emily-pc:/tmp/.ICE-unix/2241
SHELL=/usr/bin/nu
SHLVL=0
SSH_AUTH_SOCK=/run/user/1000/keyring/ssh
SYSTEMD_EXEC_PID=2026
TERM=xterm-256color
USER=emilyseville7cfg
USERNAME=emilyseville7cfg
VTE_VERSION=6800
WINDOWID=85983238
WINDOWPATH=2
XAUTHORITY=/run/user/1000/gdm/Xauthority
XDG_CONFIG_DIRS=/etc/xdg/xdg-mate:/etc/xdg
XDG_CURRENT_DESKTOP=MATE
XDG_DATA_DIRS=/usr/share/mate:/usr/share/mate:/usr/share/gnome:/usr/local/share/:/usr/share/:/var/lib/snapd/desktop
XDG_RUNTIME_DIR=/run/user/1000
XDG_SESSION_CLASS=user
XDG_SESSION_DESKTOP=mate
XDG_SESSION_TYPE=x11
XMODIFIERS=@im=ibus
_=/usr/bin/dbus-update-activation-environment
```

> @EmilySeville7cfg I am guessing that if you run the below command you get the same unescaped codes?

That's true:

![image](https://github.com/DannyBen/bashly/assets/42812113/dcdd4698-d83f-4280-b1dc-3046cb89b21c)

> What does your ruby -v say?

ruby 3.0.2p107 (2021-07-07 revision 0db68f0233) [x86_64-linux-gnu]

**Comment by EmilySeville7cfg** [comment]
> Does this following command make any difference? (added semicolons to the escape):

@DannyBen, not at all:

![image](https://github.com/DannyBen/bashly/assets/42812113/086e55ed-a58f-486b-aaf2-a183145f7bd8)

**Comment by DannyBen** [comment]
Not sure how to reproduce. I thought i might be nushell, or your Ruby - but I tested both and see the colors.

**Comment by DannyBen** [comment]
I went with the "if you can't fix it, eliminate it" approach. The prompt is now colorless, and looks like the bashly logo:
`$> bashly `

**Comment by DannyBen** [comment]
[Released in 1.0.6](https://github.com/DannyBen/bashly/releases/tag/v1.0.6).
